src/net/sourceforge/kolmafia/persistence/SkillDatabase.java

```diff
@@ -916,11 +896,15 @@ public static final long birdSkillCasts(long availableMP) {
   }
 
   public static final long stackLumpsCost() {
+    return stackLumpsCost(Preferences.getInteger("_stackLumpsUses"));
+  }
+
+  public static final long stackLumpsCost(int casts) {
     long mpCost = 1;
-    int casts = Preferences.getInteger("_stackLumpsUses");
     if (casts < 0) return mpCost;
+    if (casts > 17) return Long.MAX_VALUE;
     for (int i = 0; i <= casts; i++) {
-      mpCost += 10 * Math.pow(10, i);
+      mpCost = (10 * mpCost) + 1;
```

Comment:
The Pow formulation was suggested on the KoL wiki but this implementation actually gave an incorrect answer after 15 casts or so.  The different calculation is correct and overflows will be trapped.